Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.cpp

```diff
@@ -84,6 +85,15 @@ static bool isSupportedEncoderCodec(const StringView& codec)
     if (!isCodecAllowed)
         return false;
 
+    // Can't find restrictions for those values, but this is what Firefox/Gecko does and WPT audio-encoder-config.https.any.html checks for.
+    if (config.sampleRate < 3000 || config.sampleRate > 384000)
```

Comment:
I would be tempted to file a bug and do that as a follow-up.

Reply:
https://github.com/web-platform-tests/wpt/issues/49635